src/RegistrationController.js

```diff
@@ -139,7 +154,10 @@ function (
         },
         toJSON: function() {
           var data = Okta.Model.prototype.toJSON.apply(this, arguments);
-          return {userProfile: data};
+          return {
+            userProfile: data,
+            relayState: self.getRelayStateData()
```

Comment:
Does that mean we always return the `relatState` even it is empty?

Reply:
yeah, I had asked @iancorcoran-okta about the this.